.github/workflows/tests.yml

```diff
@@ -319,13 +361,28 @@ jobs:
           conda info -a
           conda list --show-channel-urls
 
+          echo "Installing Support Libraries"
+          pip install setuptools wheel twine
+          pip install pytest-html
+          pip install allure-pytest
+          
+          echo "Preparing Testing Report Folders on Windows"
+          mkdir -p ${{ github.workspace }}\\allure-results
+          mkdir -p ${{ github.workspace }}\\allure-results/allure-reports
+          mkdir -p ${{ github.workspace }}\\allure-results/allure-history
+          mkdir -p ${{ github.workspace }}\\tests_results
```

Comment:
No \\ on macos; only create allure-results folder, is ${{ github.workspace }} the current directory and can be omitted?